examples/prom/elliptic_eigenproblem_global_rom.cpp

```diff
@@ -458,6 +461,32 @@ int main(int argc, char *argv[])
         lobpcg->SetMassMatrix(*M);
         lobpcg->SetOperator(*A);
 
+        if (prescribe_init && (fom || (offline && id > 0)))
+        {
+            HypreParVector** snapshot_vecs = new HypreParVector*[nev];
+            for (int i = 0; i < nev; i++)
+            {
+                std::string snapshot_filename = baseName + "ref_snapshot_" + std::to_string(i);
+                std::ifstream snapshot_infile(snapshot_filename + ".0");
```

Comment:
What is the hard-coded 0?

Reply:
Change to the MPI rank. For this file, it does not use the 6 digit suffix.